mathematics_dataset/util/display.py

```diff
@@ -128,6 +130,115 @@
 
 
 # For converting ordinals to words:
+_ORD_ROOTS = [
+    'нулев',
+    'перв',
+    'втор',
+    'трет',
+    'четверт',
+    'пят',
+    'шест',
+    'седьм',
+    'восьм',
+    'девят',
+    'десят',
+    'одиннадцат',
+    'двенадцат',
+    'тринадцат',
+    'четырнадцат',
+    'пятнадцат',
+    'шестнадцат',
+    'семнадцат',
+    'восемнадцат',
+    'девятнадцат',
+]
+
+
+_ord_cases_3 = {
+    True: {
+        'male': {
+            'nomn': 'ий',
+            'datv': 'ьему',
+        },
+        'femn': {
+            'nomn': 'ья',
+            'datv': 'ьей',
+            'gent': 'ьей'
+        },
+        'mid': {
+            'nomn': 'ье',
+            'datv': 'ьему',
+        }
+    },
+    False: {
+        'nomn': 'ьи',
+        'gent': 'ьих',
+        "datv": "ьим",
+        "accs": "ьи",
+        "ablt": "ьими",
+        "loct": "ьих"
+    }
+}
+
+
+def _get_ord_cases(x, gen='male', case='nomn', is_sing=True):
+    root = _ORD_ROOTS[x]
+    end = '![Warning]'
+    if x == 3:
+        if is_sing:
+            end = _ord_cases_3[is_sing][gen][case]
+        else:
+            end = _ord_cases_3[is_sing][case]
+    elif is_sing:
+        if gen == 'male':
+            if case == 'nomn':
+                if x in (0, 2, 6, 7, 8):
+                    end = 'ой'
+                else:
+                    end = 'ый'
+            if case == 'datv':
+                end = 'ому'
+        elif gen == 'femn':
+            if case == 'nomn':
+                end = 'ая'
+            elif case in ['datv', 'gent']:
+                end = 'ой'
+        else:
+            if case == 'nomn':
+                end = 'ое'
+            elif case == 'datv':
+                end = 'ому'
+    else:
+        if case in ['nomn', 'accs']:
+            end = 'ые'
+        elif case in ['gent', 'loct']:
+            end = 'ых'
+        elif case == 'datv':
+            end = 'ым'
+        else:
+            end = 'ыми'
+
+    return root + end
+
+
+def _get_numer(x):
```

Comment:
```suggestion
def _convert_num_to_string(x):
```
или можно подумать над каким-то названием, которое более развернуто будет говорить про то. что делает функция

плюс докстринг и typing давай везде добавим

Reply:
Выдает строку для numer(al) - числителя.